OPHD/StructureManager.cpp

```diff
@@ -339,6 +339,20 @@ const StructureList& StructureManager::structureList(Structure::StructureClass s
 }
 
 
+StructureList StructureManager::allStructures()
+{
+	StructureList structuresOut;
+
+	for(auto& structures : mStructureLists)
+	{
+		auto& list = structures.second;
+		std::copy(list.begin(), list.end(), std::back_inserter(structuresOut));
+	}
```

Comment:
Should probably avoid calling the loop variable `structures` if it isn't a collection of structures. It's maybe a bit misleading, since it's actually a `std::pair`, where the `second` element is the list of structures.

We should maybe come up with a convention for naming `std::pair` objects in range-for loops. I don't object to `keyValuePair`, though it is a bit generic. Could maybe name it something more specific, like `structureClassStructuresPair`, but that gets a bit wordy.

The `list` variable could then be named `structures`, which would be a bit more descriptive.

Reply:
Fair point. I'll fix this with the below comment as well.